arbnode/transaction_streamer.go

```diff
@@ -1108,203 +828,50 @@ func (s *TransactionStreamer) writeMessages(pos arbutil.MessageIndex, messages [
 	return nil
 }
 
-// Produce and record blocks for all available messages
-func (s *TransactionStreamer) createBlocks(ctx context.Context) error {
-	s.createBlocksMutex.Lock()
-	defer s.createBlocksMutex.Unlock()
-	s.reorgMutex.RLock()
+// return value: true if should be called again
```

Comment:
This should probably say "true if should be called again immediately", as we'll still call it again after ExecuteMessageLoopDelay if it returns false